.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "env": {
+    "test": {
+      "plugins": ["@babel/plugin-transform-modules-commonjs"]
+    }
+  }
+}
```

Comment:
for module import in jest

Reply:
Jest has its own config. You can use jest config for transforming config code and then run it. Same for typescript, we have `tsconfig` which runs and transforms our code. [Jest config](https://jestjs.io/docs/en/configuration)

Reply:
got you